STANDARDS.md

```diff
@@ -79,17 +83,16 @@ Each dataset shall be versioned.
 
 * The major version shall indicate the STANDARDS that the dataset conforms to. Datasets which are not intended to conform to any STANDARDS should start with 0, e.g. `"v0.1"`. 
 
-* Datasets with versions starting with `"v1.x"` and `"v2.x"` do not follow any official STANDARDS, and thus should be considered `"v0.x"`.
+* Datasets with versions starting with `"v1.x"` and `"v2.x"` do not follow any official STANDARDS, and thus should be considered `"v0.x"`. 
 
 - A minor version change (e.g. `"v3.1"`) represents a minor addition and/or fixes problems:
-	- Adding molecules
+	- Adding molecules or removing/replacing records with an underlying problem
 	- Adding compute specifications
```

Comment:
Do we really want compute specifications to require a version bump? You already have to specifically request the compute spec you want, so I don't think there's any confusion. 

Additionally, our existing procedure for compute expansions doesn't allow modification of the dataset name, so we would have to modify or eliminate it in favor of submitting a completely new dataset for each compute spec update.